.vsts.pipelines/jobs/ci-linux_bootstrap.yml

```diff
@@ -0,0 +1,106 @@
+parameters:
+  job: null
+  matrix:
+    Production: {}
+  pool:
+    name: Hosted Ubuntu 1604
+  imageName: null
+  reportPrebuiltLeaks: false
+  dependsOn: ''
+  generatePrebuiltBurndown: false
+  runUnitTests: false
+  dockerNetwork: ''
+
+jobs:
+- job: ${{ parameters.job }}
+  dependsOn: ${{ parameters.dependsOn }}
+  strategy:
+    matrix: ${{ parameters.matrix }}
+  pool: ${{ parameters.pool }}
+  timeoutInMinutes: 270
+  variables:
+    # Prefix to distinguish artifacts from different legs.
+    artifactName: ${{ format('{0} $(type)', parameters.job) }}
+    # Use ":z" to set selinux flag for sharing in build-owned root dir. https://docs.docker.com/storage/bind-mounts/#configure-the-selinux-label
+    docker.agentSrc.map: -v $(Build.SourcesDirectory):/agentSrc:z
+    docker.agentSrc.work: -w /agentSrc
+    docker.drop.map: -v $(dropDirectory):/drop:z
+    docker.logs.map: -v $(dropDirectory)/logs:/logs:z
+    docker.root.map: -v $(rootDirectory):/root:z
+    docker.run: docker run --rm
+    docker.src.map: -v $(rootDirectory)/sb/source-build:/src:z
+    docker.src.work: -w /src
+    docker.tb.map: -v $(rootDirectory)/sb/tarball:/tb:z
+    docker.tb.work: -w /tb
+    docker.bst.map: -v $(System.ArtifactsDirectory)/boostrap_root:/bst:z
+    docker.bst.work: -w /bst
+    dropDirectory: $(stagingDirectory)/drop
+    imageName: ${{ parameters.imageName }}
+    reportPrebuiltLeaks: ${{ parameters.reportPrebuiltLeaks }}
+    generatePrebuiltBurndown: ${{ parameters.generatePrebuiltBurndown }}
+    rootDirectory: $(Build.SourcesDirectory)/..
+    stagingDirectory: $(rootDirectory)/sb/staging
+    tarballName: tarball_$(Build.BuildId)
+    SOURCE_BUILD_SKIP_SUBMODULE_CHECK: true
+    # Default type, can be overridden by matrix legs.
+    type: Production
+
+  steps:
+  # - template: ../steps/cleanup-unneeded-files.yml
+  - template: ../steps/docker-cleanup-linux.yml
+  - template: ../steps/calculate-config-flags.yml
```

Comment:
Is this template necessary?

Reply:
I think cleanup could be helpful. Other two, not so much. Ill prune them.